db/migrate/20171102224754_add_role_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleToUsers < ActiveRecord::Migration[5.1]
+  def change
+    add_column :users, :role, :integer, default: 0
```

Comment:
I was about to comment in the first migration that we needed to add a role, but it's added here. Nice!